### DISCUSSION TITLE
Feature request: Usage data as feed

### DISCUSSION THREAD

**Discussion by greystate** [umbraco/Umbraco.Cloud.Issues#811] [Ideas]
### Issue description

Is the data on the Usage dashboard available in a feed of some sort? (e.g. an API call?)

Just as some people are using **Uptime Robot** or similar services for monitoring the sites' availability, it would be nice to be able to see when a site is close to its limits for e.g. Content nodes etc.

**Comment by sajumb** [comment]
Currently, there is no public API endpoint for getting this usage information. But we understand that it good be very beneficial for monitoring purposes. We are currently considered various improvements to show the usage in another context. e.g. on the projects overview or show historic data on the usage page. A public endpoint for getting this information is likewise a very good feature to offer.

I have added the request to backlog and will update this post when we have more offering regarding usage.
